src/components/Dropdown/DropdownInput.tsx

```diff
@@ -80,7 +86,7 @@ export function DropdownInput(props: {
       id={screenReaderUUID && generateDropdownId(screenReaderUUID, -1)}
       autoComplete='off'
       aria-describedby={screenReaderUUID}
-      aria-activedescendant={screenReaderUUID && generateDropdownId(screenReaderUUID, focusedIndex)}
+      aria-activedescendant={!isTyping ? (generateDropdownId(screenReaderUUID, focusedIndex)) : ''}
```

Comment:
small nit but can we flip the boolean to just `isTyping ? generateDropdownId : ''`? a negated boolean is very slightly harder to read